server/Game.js

```diff
@@ -4,15 +4,33 @@ import { PING_TIMEOUT, SELECT_WORD_TIMEOUT, TICK_TIMEOUT } from '../shared/const
 import protocol from '../shared/protocol';
 import createHistory from './History';
 
-const words = require(`../${process.env.WORDS_FILE}`);
+const words = [...new Set(require(`../${process.env.WORDS_FILE}`))].filter(Boolean);
+
+if(words.length < 3) {
```

Comment:
nit: the style in this project has a space after `if` / `while` etc, so `if (words.length < 3) {` for example. Same in the rest of this PR :-)